dali/kernels/imgproc/resample/resampling_impl_cpu.h

```diff
@@ -166,9 +167,9 @@ inline void ResampleHorz(Surface2D<Out> out, Surface2D<In> in,
 template <typename Out, typename In>
 inline void ResampleAxis(Surface2D<Out> out, Surface2D<In> in,
                          const int *in_indices, const float *coeffs, int support, int axis) {
-  if (axis == 0)
+  if (axis == 1)
```

Comment:
Why axes are reverted?

Reply:
I switched to vector-like indexing, so that each new dimension (Y, Z, etc) are given new indices and existing ones have constant indices. This way X has always index 0, regardless of dimensionality. Tensor shape indices (starting from outermost) do not have this property.

Reply:
Understood